ompi/mpi/fortran/mpif-h/session_finalize_f.c

```diff
@@ -80,4 +80,11 @@ void ompi_session_finalize_f(MPI_Fint *session, MPI_Fint *ierr)
 
     c_ierr = PMPI_Session_finalize(&c_session);
     if (NULL != ierr) *ierr = OMPI_INT_2_FINT(c_ierr);
+
+  /* This value comes from the MPI_SESSION_NULL value in mpif.h.  Do not
+     change without consulting mpif.h! */
+
+  if (MPI_SUCCESS == c_ierr) {
+      *session = 0;
+  }
```

Comment:
Can you fix the indenting here?  We use 4-space tabs in Open MPI.  Thanks!

Reply:
Yes, I don't know how that happened. It's fixed now, thanks for noticing!